### PR TITLE
Add map for rhcos

### PR DESCRIPTION
**Pull request by ravanelli**
Signed-off-by: Renata Ravanelli <rravanel@redhat.com>

### CODE REVIEW

**Comment by ravanelli** [comment]
@darkmuggle @jlebon Would mind in checking it?
It is part of https://github.com/openshift/release/pull/19679 as well.

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -520,6 +520,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/coreos/**/*-rhcos*.yaml:
```
Comment (jlebon) [inline core-services/prow/02_config/_plugins.yaml:523]: I only barely understand this file, but offhand I'm not sure why we wouldn't just include all YAML files (i.e. `ci-operator/config/coreos/**/*.yaml`). And similarly below.

**Review by jlebon — APPROVED** [review]
Seems sane to me!

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (ravanelli) [reply core-services/prow/02_config/_plugins.yaml:523]: @jlebon I was trying to not mess or touch other things I didn't add in the configs/jobs hehe 🙃. It shouldn't be a problem though.

**Review by darkmuggle — APPROVED** [review]
/lgtm

**Review by petr-muller — APPROVED** [review]

**Pull request merged**